src/sortStudents.ts

```diff
@@ -1,16 +1,81 @@
 
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades',
 }
 
 // create SortOrder type
-export type SortOrder;
+export type SortOrder = 'desc' | 'asc';
 
+function findAverage(arr:number[]):number {
+  return arr.reduce((sum, num) => (sum + num)) / arr.length;
+}
+
+export function sortStudents(
+  students:Student[],
+  sortBy:SortType,
+  order:SortOrder,
+  ):object[] | number {  // eslint-disable-line
```

Comment:
```suggestion
  ): Student[] | number {  // eslint-disable-line
```
I don't know why you wrote 'number' here, but about object - you can use your custom type

Reply:
Oh, I see why you use 'number' here :)